src/main/java/org/jenkinsci/plugins/fodupload/SharedPollingBuildStep.java

```diff
@@ -92,9 +93,11 @@ public static FormValidation doCheckPollingInterval(String pollingInterval) {
 
     // Form validation
     @SuppressFBWarnings("NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE")
+    @POST
     public static FormValidation doTestPersonalAccessTokenConnection(final String username,
                                                                      final String personalAccessToken,
                                                                      final String tenantId) {
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);

```

Comment:
Is this necessary to fix this vuln?

Reply:
The original vuln summary from Jenkins security team stated : "@POST and Jenkins.getInstance().checkPermission(Jenkins.ADMINISTER); not added when there is a validation check on the form." So it would seem to be necessary.